kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -103,6 +103,7 @@
       <ul class="ul-reset">
         <template v-for="(facility, idx) in facilities">
           <li :key="idx">
+            <KLabeledIcon icon="facility" />
```

Comment:
The `KExternalLink` actually supports an `icon` prop, so I think you can just add something like `icon="facility"` as a prop. See this code

https://github.com/learningequality/kolibri-design-system/blob/v0.2.x/lib/buttons-and-links/KExternalLink.vue